### PR TITLE
PEP 581 Updates

### PR DESCRIPTION
**Pull request by Mariatta**
- include recent changes to bpo
- clarify the downsides of GitHub

### CODE REVIEW

File: pep-0581.rst [hunk 1]
```diff
@@ -152,6 +166,50 @@ I believe the effort of creating and maintaining GitHub integrations and bots
...
+  risk of vendor lock-in. However, this is an existing problem since CPython's
+  codebase is already on GitHub. This is also not a unique problem to CPython.
+  As a precautionary step, CPython's repository on GitHub has
+  been backed up on daily basis since June 2018. [#]_
+
+- Bots maintenance costs money, and also takes up volunteer time. We would
+  be shifting maintenance burden from Roundup to the bots. At the very least,
+  so far we've been able to address any bugs/issues related to the bots/GitHub
+  APIs rather quickly, in a matter of days, instead of months or years. GitHub
+  APIs are extensive, and used not by CPython's bots but also by the wider
```
Comment (gvanrossum) [inline pep-0581.rst]: not only

File: pep-0581.rst [hunk 2]
```diff
@@ -152,6 +166,50 @@ I believe the effort of creating and maintaining GitHub integrations and bots
...
+- Using GitHub could make it easier for people to post disruptive or spammy comments.
+  It is true that there has been incidents where core developers have to moderate
+  and lock disruptive discussions on GitHub. Thankfully, GitHub interface makes
+  it easy for core developers to moderate discussions. In addition, incidents
+  can be escalated to GitHub.
+
+- Manually editing issue templates can be cumbersome and error-prone. However,
+  for most people, creating issues on GitHub will be a much better experience
+  than creating issue on bpo. The numerous fields and textbox to choose from
+  can be confusing and intimidating to a newcomer, and it was not possible
```
Comment (gvanrossum) [inline pep-0581.rst]: was -> is? Or did bpo add the ability to edit messages?

File: pep-0581.rst (hunk 1 above)
Comment (Mariatta) [reply pep-0581.rst]: Thanks. Updated.

File: pep-0581.rst (hunk 2 above)
Comment (Mariatta) [reply pep-0581.rst]: "is". I've updated the text. Thanks.

**Comment by gvanrossum** [comment]
Thanks, LGTM.

**Comment by Mariatta** [comment]
Thanks!

**Pull request merged**